scraper.py

```diff
@@ -31,6 +31,8 @@
     if doctors_result:
         foreign_doctors_scraper.doc_id = len(doctors_result)
         foreign_docs_result = foreign_doctors_scraper.run_scraper()
+    else:  # in the event doctor scraper fails, attempt on foreign doctors with id 1
+        foreign_docs_result = foreign_doctors_scraper.run_scraper()
```

Comment:
Nope. The foreign doctors and local are combined. We can't have one without the other.